language-design.md

```diff
@@ -17,7 +17,11 @@ For each quote, describe what it was about the quote that led you pick it.
 
 **Response**
 
----
+In the Pavlus article, they include a quote from Andreas Stefik: "I think that whenever you make a product design simpler there’s a potential danger of removing features that experts need" [Pavlus, 2012]. This quote stood out to me because while I do beleive that it is a valid design concern, it could be read as in conversation with broader discussions about computing access and how knowing, or not knowing, specific languages are used as ways to gatekeep technical fields and communities. This is discussed a lot more in the Yang and Rabkin article about language stereotypes [Yang and Rabkin, 2015]. I wonder if when Stefik says a danger of "removing features that experts need" that it is also a future way for people to judge those who know Quorum as an "inferior" language as a veiled way to judge the programers themselves.
+
+"In stead of designing a thing, you need to design a way of doing. And this way of doing must make some choices now but leave other choices to a later time." [Steele, 1998] This is definitely a perspective I want to remember. In theory, I agree that users should have control and voice during the design process - they are after all, the ones who are going to use it at the end. In practice, that feels really hard to do, but remembering this idea might help. It doesn't sound easier, but perhaps will ultimately be more useful and live longer than I do.
+
+"Obey the principle of least astonishment." [Bloch, 2006] I picked this quote because it's so simple. Name, define, and design things so that users aren't surprised. Make it predictable, or at least as predicable as possible so that there are less bugs, users are less confused, and so that users are less frustrated. I'd want there to be excitement, and joy, but I personally hate the moments where I say "why is done like that" and would want to design something better for my users. Though, I suppose like my previous quote, it's probably easier said that done.
```

Comment:
I agree! There are times to get creative, but this is definitely not one of the times. There's sometimes difficulty in being concise, though; sometimes, you're afraid that you're not getting across every point so you end up with something horrendous like "getSecondToLastValueOfObjectAt......"